ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/HomeScreenMapContainerViewModel.kt

```diff
@@ -68,7 +72,7 @@ internal constructor(
     surveyRepository
   ) {
 
-  val mapLocationOfInterestFeatures: LiveData<Set<Feature>>
+  val mapLocationOfInterestFeatures: Flow<Set<Feature>>
```

Comment:
Since this represents UI state, shouldn't it be modeled as a `StateFlow` so that the last value is replayed to late subscribers?

Reply:
Thanks! Done